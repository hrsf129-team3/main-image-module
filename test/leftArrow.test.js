import React from 'react';
import LeftArrow from '../client/src/components/LeftArrow.jsx';

describe('Left Arrow', () => {
  it('should render left arrow component', () => {
    const component = shallow(<LeftArrow />);
    expect(component).toMatchSnapshot();
  });
})


