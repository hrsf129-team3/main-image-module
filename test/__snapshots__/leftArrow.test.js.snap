// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Left Arrow should render left arrow component 1`] = `ShallowWrapper {}`;
